api/src/Entity/ContentType.php

```diff
@@ -30,4 +29,14 @@ class ContentType extends BaseEntity {
      * @ORM\Column(type="json", nullable=true)
      */
     public ?array $jsonConfig = [];
+
+    public function getConfig(?string $key = null) {
+        if (null != $this->jsonConfig) {
+            if (null != $key) {
+                return $this->jsonConfig[$key];
+            }
+        }
+
+        return $this->jsonConfig;
+    }
```

Comment:
Das Argument dieser Methode wird bisher nur in einem Test verwendet, sonst nirgends. Ich würde diesen Getter löschen und im Test den Eintrag unter $key selber aus der JSON Config fischen.